src/components/TodoItem/TodoItem.js

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cn from 'classnames';
+
+export const TodoItem = (props) => {
+  const { title, id, completed, onDeleteTodo, onUpdateCompleted } = props;
+
+  return (
+    <li className={cn({ completed })}>
+      <div className="view">
+        <input
+          type="checkbox"
+          className="toggle"
+          id={id}
+          onChange={onUpdateCompleted}
+          checked={completed ? 'checked' : ''}
```

Comment:
use cn

Reply:
Not sure what you mean. How to accomplish that?